ert_logger/__init__.py

```diff
@@ -1,4 +1,5 @@
-import os, pwd
```

Comment:
Not related to the commit text

Reply:
I think this was the doing of isort (which I know we don't use)